docs/appendices/release-notes/5.5.1.rst

```diff
@@ -65,3 +65,20 @@ Fixes
 - Fixed an issue that caused a ``ColumnUnknownException`` to be thrown when
   attempting to drop a column with a :ref:`check_constraint`. All relevant
   constraints are dropped together with the column.
+
+- Fixed an issue that caused the hash-join operator generate invalid hashes which
+  lead to a broken join operation when there are more than two relations
+  involved e.g.::
+
+    SELECT * FROM t1,t2,t3 WHERE t3.c = t1.a AND t3.c = t2.b AND t1.a = t2.b;
+
+  will generate the logical plan::
```

Comment:
I think past tense it's better given the whole past-tense phrasing of the explanation before:
```suggestion
  would generate the logical plan::
```